layouts/partials/head.html

```diff
@@ -46,5 +46,5 @@
 {{ partial "hooks/head-end.html" . }}
 <!--To comply with GDPR, cookie consent scripts places in head-end must execute before Google Analytics is enabled -->
 {{ if eq (getenv "HUGO_ENV") "production" }}
-{{ template "_internal/google_analytics_async.html" . }}
+{{ template "_internal/google_analytics.html" . }}
```

Comment:
I'm not sure if this matters, as we don't have an `_internal` subdirectory in our repo. (I probably copied this from somewhere as well.)